test/Microsoft.AspNet.Mvc.ModelBinding.Test/Binders/KeyValuePairModelBinderTest.cs

```diff
@@ -28,8 +26,7 @@ public class KeyValuePairModelBinderTest
             var retVal = await binder.BindModelAsync(bindingContext);
 
             // Assert
-            Assert.NotNull(retVal);
-            Assert.Null(retVal.Model);
+            Assert.Null(retVal);
```

Comment:
I don't get why this case is a `null` with errors. We've found part of the model - at this point we've commited to binding using whatever prefix is currently in use.